.changeset/chatty-boxes-judge.md

```diff
@@ -0,0 +1,6 @@
+---
+"@cambly/syntax-core": major
```

Comment:
@somethiiing should this just be a minor change?

Reply:
yeah, major changes are for things that are breaking. such as removing a prop, or changing a propname, etc. add-ons or not breaking changes should all be minor

Reply:
got it!